package.json

```diff
@@ -13,15 +13,21 @@
   "devDependencies": {
     "@algolia/client-search": "4.9.1",
     "@babel/cli": "7.13.0",
+    "@babel/plugin-transform-react-jsx": "^7.13.12",
```

Comment:
This dep should be pinned.

Reply:
Done in https://github.com/algolia/react-recommendations/pull/6/commits/47398ad5fd3e21ff8e9edad6b2a0ea0eeb4b9444